packages/discovery-react-components/src/components/SearchResults/components/Result/Result.tsx

```diff
@@ -102,26 +101,32 @@ export const Result: React.FunctionComponent<ResultProps> = ({
     fetchDocumentsResponseStore: { isLoading }
   } = useContext(SearchContext);
 
-  const firstPassage: DiscoveryV2.QueryResultPassage | undefined = get(
+  const passages: DiscoveryV2.QueryResultPassage[] | undefined = get(
     result,
-    'document_passages[0]'
+    'document_passages'
+  )?.filter(passage => !!get(passage, 'passage_text'));
+  const passageTexts: (string | undefined)[] | undefined = passages?.map(passage =>
+    get(passage, 'passage_text')
   );
-  const firstPassageText = get(firstPassage, 'passage_text');
-  const hasPassage = usePassages && !!firstPassageText;
-  let displayedText: string | undefined = get(result, bodyField);
+  const hasPassage = usePassages && !!passageTexts && passageTexts.length > 0;
```

Comment:
Nit, since you're already updating the singular vars to plural
```suggestion
  const hasPassages = usePassages && !!passageTexts && passageTexts.length > 0;
```